### PR TITLE
a redis scan match key bug

### PR DESCRIPTION
**Pull request by ucanme**
redis scan match key always return array[1] as  results, but it's length maybe 0

### CODE REVIEW

**Comment by ucanme** [comment]
hello, please check and accpet.

File: persistence/session/redis/store.go [hunk 1]
```diff
@@ -111,6 +111,7 @@ func (s *Store) Iterate(fn session.IterateFn) error {
 		}
 		if len(arr) >= 1 {
 			for _, v := range arr[1:] {
+				if len( v.([]interface{})) > 1 {
```
Comment (DrmagicE) [inline persistence/session/redis/store.go:114]: It is not necessary to add the if statement.  If the length is 0, the `for range` part just skip it. 
Why do you want to add this statement？ Did you find some bugs?
BTW, it is recommended to run `go fmt` to format your code before making a PR

**Comment by DrmagicE** [comment]
@ucanme Hi, would you please provide more details about how to reproduce the problem？

**Comment by DrmagicE** [comment]
close due to inactive

**Pull request closed without merging**